nginx/nginx.conf

```diff
@@ -25,7 +25,9 @@ http {
 
     keepalive_timeout  65;
 
-    #gzip  on;
+    gzip on;  
```

Comment:
If our intentions are to contribute to the repo upstream, we may want to make this an option via Environment variables, instead of enabling it explicitly. Just make the default the current setting.